libflux/go/libflux/parser.go

```diff
@@ -76,5 +77,5 @@ func ParseIntoFbs(s string) *ast.Package {
 	defer C.free(unsafe.Pointer(ptr))
 
 	data := C.GoBytes(ptr.data, C.int(ptr.len))
-	return ast.Package{}.FromBuf(data)
+	return ast.Package{}.FromBuf(data, flatbuffers.UOffsetT(C.int(ptr.offset)))
```

Comment:
What is this line of code doing? I see a couple of offsets here, but its hard to tell.

Reply:
I think it's taking the offset field of `ptr`, typecasting it to a C integer (`C.int(...)`) and then typecasting that to a `flatbuffers.UOffsetT`.  I'm not sure why we need the intermediate cast to a C integer though.